src/obfuscation.ts

```diff
@@ -9,13 +9,6 @@ let getRandomValues: (length: number) => Uint8Array;
 if (typeof window !== 'undefined' && window.crypto) {
   // Browser environment
   getRandomValues = (length: number) => window.crypto.getRandomValues(new Uint8Array(length));
-} else if (typeof navigator !== 'undefined' && navigator.product === 'ReactNative') {
-  // React Native environment
-  require('react-native-get-random-values');
-  getRandomValues = (length: number) => {
-    const array = new Uint8Array(length);
-    return window.crypto.getRandomValues(array);
-  };
```

Comment:
can this be kept and the dependency removed from `package.json` ?

Reply:
alternatively - how would I provide this functionality in react native?

Reply:
We only need `getRandomValues` in node for generating the obfuscated configuration wire. The purpose of this code isn't really about functionality, but making common compatible with these other environments

Reply:
I feel like it might be possible to adjust the `getPrecomputedAssignments` to also accept a `getRandomValues` argument maybe so we don't do this in the common sdk

Reply:
Ok cool if only needed on the generation side, not unpacking in offline mode for clients, then fine to omit. People might use react native with nodejs instead of our servers.